src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -119,6 +119,41 @@ function _assignPanelId(element) {
   }
 }
 
+/**
+ * Check and warns if element has conflicting attributes.
+ * @param element Root element to check
+ * @param attribute An attribute that is conflicting with other attributes
+ * @param attrsConflictingWith The attributes conflicting with `attribute`
+ */
+function _warnConflictingAttributes(element, attribute, attrsConflictingWith) {
+  if (!(attribute in element.attribs)) {
+    return;
+  }
+  for (let i = 0; i < attrsConflictingWith.length; i += 1) {
```

Comment:
Can we use `i++`?

Reply:
We have [no-plusplus](https://eslint.org/docs/rules/no-plusplus) in our linter.

Reply:
What do you think of using `attrsConflictingWith.forEach` instead then?